1-mandatory/2-people-i-know.js

```diff
@@ -378,22 +378,46 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+for (var i = 0; i < friends.length; i++){
+  if (friends[i].age >= 35){
+    thirtyFiveOrOlder.push(friends[i]);
+    // thirtyFiveOrOlder.push(friends[i].name.first);
+  }
+};
 
+console.log(thirtyFiveOrOlder);
 /*
 3) Find the email address
 Next, I want you to find all of my friends who work for "POWERNET" and then store their emails in the array below
 */
 
 let powerNetEmails = [];
+for (var i = 0; i < friends.length; i++){
+  if (friends[i].company == "POWERNET") {
+    powerNetEmails.push(friends[i].email);
+  }
+};
+console.log(powerNetEmails);
 
```

Comment:
Consider: You could also do it with the use of filter and map methods
friends.filter(friend => friend.company === "POWERNET").map(friend => friend.email);